pycoal/mineral.py

```diff
@@ -303,9 +305,33 @@ def __init__(self, algorithm=SAM, **kwargs):
             **kwargs: arguments that will be passed to the chosen classifier
         """
 
-        # set the user's chosen classifier 
-        self.algorithm = algorithm
-
+        # parse config file
+        config = configparser.ConfigParser()
+        
+        try:
+            with open('config.ini') as config_file:
+                config.read_file(config_file)
+        except IOError:
+            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), 'config.ini')
+
+        set_algo = None
```

Comment:
These should not be `None` they should be set to the default values.

Reply:
The algorithm is set as default above. This being set to None allows me not to update self. algorithm, leaving it as default. The set_algo is just for the value pulled from the config file